pylti/tests/test_flask.py

```diff
@@ -454,12 +460,18 @@ def request_callback(request, cburi, headers):
         self.assertFalse(self.has_exception())
         self.assertEqual(ret.data, "grade=False")
 
+    def request_callback(self, request, cburi, headers):
```

Comment:
I think the right way to do this would be:

```
def mock_request_callback_func():
  def request_callback(request, cburi, headers):
    return 200, headers, self.expected_response
  return request_callback
```

```
httpretty.register_uri(httpretty.PUT, uri, body=self.mock_request_callback_func())
```